docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Docs Directory
+
+## How-Tos
+* [Dev Of The Week Zendesk](dev-of-the-week-zendesk.md) - How to find stuff in zendesk if you are dev of the week
+* [Dropbox Share Guide](dropbox-share-guide.md) - How to add new dropbox shares to staging
+* [Honeybadger](honeybadger.md) - How to use honeybadger to triage site issues
+* [Importing Data into Pegasus](importing-data.md)
+* [Instrumenting with Google Analytics](instrumenting-analytics.md)
+* [Logging](logging.md) - How to do various things with dashboard and pegasus logs
```

Comment:
oh, I saw that you were the last person to touch the file so I thought you might know. But I guess you were only fixing a typo. I'll just leave it.